lib/api.rb

```diff
@@ -208,6 +223,8 @@ def validation_error(msg)
       u.password_hash = params[:masterpasswordhash]
       u.password_hint = params[:masterpasswordhint]
       u.key = params[:key]
+      u.public_key = params[:keys][:publicKey]
+      u.private_key = params[:keys][:encryptedPrivateKey]
```

Comment:
These probably require validation.

Reply:
Ok for the privatekey. But for the public key, hard to find the regex.